libraries/rush-lib/src/logic/pnpm/PnpmOptionsConfiguration.ts

```diff
@@ -425,6 +443,7 @@ export class PnpmOptionsConfiguration extends PackageManagerOptionsConfiguration
     this._globalPatchedDependencies = json.globalPatchedDependencies;
     this.resolutionMode = json.resolutionMode;
     this.autoInstallPeers = json.autoInstallPeers;
+    this.minimumReleaseAge = json.minimumReleaseAge;
```

Comment:
Similar to `globalIgnoredOptionalDependencies`, we should emit a warning if the version of PNPM is too old to use this feature. See https://github.com/microsoft/rushstack/blob/3514f6cdf11a866b7a53468ebf5e3aa0452955af/libraries/rush-lib/src/logic/installManager/InstallHelpers.ts#L77-L93